homeassistant/components/harmony/remote.py

```diff
@@ -207,6 +217,7 @@ def device_info(self):
             ),
             "name": self.name,
             "model": model,
+            "protocol": self._client.protocol,
```

Comment:
https://developers.home-assistant.io/docs/device_registry_index/#device-properties

Protocol isn't a valid key here

Reply:
Ah, that's where it is used for. Removed. :-)